### PR TITLE
Update privacy policy: new blog platform Ghost

### PR DESCRIPTION
**Pull request by philipecosia**
Update to the privacy policy section 6b. Blog functions to reflect switch from Tumblr to Ghost.

### CODE REVIEW

File: en.md [hunk 1]
```diff
@@ -72,15 +72,9 @@ Microsoft/Bing is certified under the “Privacy Shield” US-EU data protection
 
 We provide a blog in which we publish various posts on topics about our activities in order to circulate our website and its contents on the internet and to increase our visitor numbers. 
 
-We use the “Tumblr” blogging platform to operate the blog. The legal basis for this processing of your personal data is Art. 6 Abs. 1 lit. f of the General Data Protection Regulation. Tumblr is operated by Tumblr Inc., 35 East 21st St, Ground Floor, New York, NY 10010, USA.
+We use the “Ghost” blogging platform to operate the blog. Ghost does not have access to any of your personal information. For information on data protection with this third-party supplier, go to: https://ghost.org/privacy/.
```
Comment (DoHe) [inline en.md]: Wasn't IP a personal information? They do have access to that obviously.

**Comment by philipecosia** [comment]
Important: This is only to be merged once a) reviewed b) ghost is live for users and c) the German and French version are also updated.

@DoHe Could you help me out with this. From what I can tell looking at Ghost's privacy policy and their DPA even if we don't host it ourselves in the beginning they don't have access to our users personal information. https://ghost.org/privacy/ https://ghost.org/dpa/ But I'm not sure.

File: en.md (hunk 1 above)
Comment (philipecosia) [reply en.md]: @DoHe Good point. I updated it.

**Comment by DoHe** [comment]
They seem to mostly talk about data they themselves gather from us, not so much about the data they might collect when hosting a blog.

**Comment by DoHe** [comment]
@philipecosia you can just post a comment and mention me if you want me to review this again

File: fr.md [hunk 2]
```diff
@@ -231,4 +226,4 @@ Pour faire valoir une opposition ou une révocation, vous pouvez contacter notre
 
 ---
 
-_Date de la déclaration sur la protection des données : 08/05/2018_
```
Comment (LostCrew) [inline fr.md:234]: Do we want to automatize this? A minuscole and inobtrusive solution could be found

**Review by knee-guts — APPROVED** [review]
🔌

**Pull request merged**